### PR TITLE
Documentation: specify how WindowDialog's default title bar is made

### LINKED ISSUE #40828: Default WindowDialog title bar
**Godot version:**

3.2.3

**Issue description:**

So I just wasted some time trying to figure out how to do a proper title bar in WindowDialog. Turns out the `panel` style is used both for contents and title bar. Well, this is mentioned in the class ref, but what isn't mentioned is how the default WindowDialog is made. **The trick is that title bar is made with a top border and an expand margin in the panel stylebox.** Would be probably useful if the docs mentioned that.

### PR DESCRIPTION
**Pull request by hhsdev**
Fixes #40828

### CODE REVIEW

File: doc/classes/WindowDialog.xml [hunk 1]
```diff
@@ -41,7 +41,7 @@
 			The vertical offset of the close button.
 		</theme_item>
 		<theme_item name="panel" type="StyleBox">
-			The style for both the content background of the [WindowDialog] and the title bar.
+      The style for both the content background of the [WindowDialog] and the title bar. The title bar is created with a top border and an expand margin of [code]panel[/code].
```
Comment (NathanLovato) [inline doc/classes/WindowDialog.xml:44]: Here's a suggestion I feel makes the description read a little more clearly in plain English. Also, it restores the tabs for indentation, that were changed into spaces.

```suggestion
			The style for both the content background of the [WindowDialog] and the title bar. The title bar is created with a top border and an expand margin using the [code]panel[/code] stylebox.
```

**Review by NathanLovato — CHANGES_REQUESTED** [review]
Thanks for your contribution, I left you a suggestion.

**Review by NathanLovato — APPROVED** [review]

**Comment by akien-mga** [comment]
Could you squash the two commits into one? See [PR workflow](https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#the-interactive-rebase).

**Comment by akien-mga** [comment]
Thanks!
